eslint.config.mjs

```diff
@@ -47,7 +48,11 @@ export default tseslint.config(
     reactRecommended,
     reactJsxRuntime,
     {
-        rules: geminiTesting.rules,
+        rules: _.omit(geminiTesting.rules, [
+            // Rules that conflict with typescript-eslint.
+            "no-undef",
+            "no-unused-vars",
```

Comment:
Because it conflicts with typescript-eslint, as stated in the comment above. It provides its own rule, `@typescript-eslint/no-unusd-vars`, because the original one doesn't work well with TS code.